src/graphql/models/departmentModel.ts

```diff
@@ -0,0 +1,26 @@
+import { ObjectType, Field, InputType } from "type-graphql";
+import { EmployeeModel } from "./employeeModel";
+
+@ObjectType()
+export class DepartmentModel {
+	@Field({ nullable: false })
+	departmentId: number;
+
+	@Field({ nullable: true })
+	departmentName: string;
+
+	@Field(() => [EmployeeModel], { nullable: true })
+	Employee: [EmployeeModel];
```

Comment:
If this is returning multiple employees then the field name must be plural. Also should be "employees" not uppercase "Employees"

Reply:
The changes has been fixed in the lastest commit